coala_quickstart/Strings.py

```diff
@@ -50,3 +50,15 @@
 
 You can see all of them here: {}
 """.format(BEAR_DOCS_URL)
+
+PRINT_BEARS = {'unusable':
+               {'msg': """
+\nBased on the configuration options the following
```

Comment:
remove the `\n` in the front of the messages as it is creating additional blank lines